setup.py

```diff
@@ -44,7 +44,7 @@
     # Used for oscar.test.newfactories
     'factory-boy>=2.4.1,<3.0',
     # Used for automatically building larger HTML tables
-    'django-tables2>=1.0.4,<1.1',
+    'django-tables2==1.2.6',
```

Comment:
We shouldn't pin a specific version of dependencies. I think its better to do 'django-tables2>=1.2,<1.3'